src/constants/task.ts

```diff
@@ -0,0 +1,33 @@
+export enum TaskOperation {
+  ADD = 'Add',
+  EDIT = 'Edit',
+}
+
+export const TaskValidationLimits = {
+  TITLE_MIN_LENGTH: 3,
+  DESCRIPTION_MIN_LENGTH: 10,
+};
+
+export const TaskValidationMessages = {
```

Comment:
Generally - Locale files are used to store user facing messages. Based on the set locale (example - en - english), the respective message is then shown.

Do we have locale support in this app? If yes - We should use that instead. If not - We need add locale support first then.

Reply:
Yeah, we do have locale support in the app. Added these validation messages to the local file.